sktime/classifiers/tests/test_columnensembler.py

```diff
@@ -0,0 +1,82 @@
+__author__ = 'Aaron Bostrom'
+
+from sktime.pipeline import FeatureUnion, Pipeline
+from sklearn.tree import DecisionTreeClassifier
+from sktime.transformers.series_to_series import RandomIntervalSegmenter
+from sktime.transformers.series_to_tabular import RandomIntervalFeatureExtractor
+from sklearn.preprocessing import FunctionTransformer
+from sktime.transformers.compose import RowwiseTransformer
+
+import numpy as np
+
+from sklearn.neighbors.classification import KNeighborsClassifier
+from sktime.classifiers.time_series_neighbors import KNeighborsTimeSeriesClassifier as KNNTSC
+from sktime.datasets.base import _load_dataset
+from sktime.dictionary_based.boss import BOSSEnsemble
+from sktime.contrib.column_ensembler import ColumnEnsembler, SimpleColumnEnsembler
+
+
+def test_univariate_column_ensembler():
+    ct = ColumnEnsembler(
+    [("KNN1", KNNTSC(n_neighbors=1), [1]),
+    ("KNN2", KNNTSC(n_neighbors=1), [2])]
+    )
+
+
+def test_simple_column_ensembler():
+    X_train ,y_train  = _load_dataset("JapaneseVowels", "TRAIN", True)
+    X_test, y_test = _load_dataset("JapaneseVowels", "TEST", True)
+
+    cts = SimpleColumnEnsembler(KNNTSC(n_neighbors=1))
+
+    cts.fit(X_train, y_train)
+    print(cts.score(X_train, y_train))
```

Comment:
would remove print statement here and throughout the test file and replace it with some sensible assertion, e.g. checking whether shape of y_pred matches shape of y_test or comparing ColumnEnsembler results with manual ensembling